pkg/resources/prometheusRules.go

```diff
@@ -62,29 +62,6 @@ func (r *AlertReconcilerImpl) ReconcileAlerts(ctx context.Context, client k8scli
 			r.Log.Infof("Operation result", l.Fields{"productName": r.ProductName, "alertName": alert.AlertName, "result": string(or)})
 		}
 	}
-
-	rule := &monitoringv1.PrometheusRule{}
-	err := client.Get(ctx, k8sclient.ObjectKey{
-		Name:      "ksm-monitoring-alerts",
-		Namespace: "redhat-rhoam-middleware-monitoring-operator",
-	}, rule)
-	if !errors.IsNotFound(err) {
-		if err := client.Delete(ctx, rule); err != nil {
-			r.Log.Info("Alert \"ksm-monitoring-alerts\" was deleted")
-		}
-	}
-
-	rule = &monitoringv1.PrometheusRule{}
-	err = client.Get(ctx, k8sclient.ObjectKey{
-		Name:      "rate-limit-soft-limits",
-		Namespace: "redhat-rhoam-marin3r",
-	}, rule)
-	if !errors.IsNotFound(err) {
-		if err := client.Delete(ctx, rule); err != nil {
-			r.Log.Info("Alert \"rate-limit-soft-limits\" was deleted")
-		}
-	}
-
```

Comment:
General question, should this be only code merged once 1.7.0 has been released to all production clusters or we good to just merged into 1.8.0 release ? I'm just thinking that if not all production environments are on 1.7.0 and say a patch version was made that skipped that version and went straight to 1.8.0, these alerts may still be kept on those clusters with this change :thinking:

Reply:
If I'm not wrong, the only way to skip 1.7.0 is ti have 1.8.0 to mark it as `skipped` or something :thinking:

Reply:
So as @makslion said the only that a customer could install 1.8.0 with out installing 1.7.0 is if 1.8.0 is marked to skip 1.7.0. All production clusters will have to install 1.7.0 first. But it is not a bad question to ask release coordinator just to confirm that confirm the 1.8.0 release will not be skipping the 1.7.0 release. 

Max you should ask that question on the jira. If you need help finding the release coordinator I can help with that.

Reply:
It was confirmed that 1.7.x will not be skipped

Reply:
Great, thanks, was just wondering in case it's something to consider :+1: